### PR TITLE
Configuring Che installation on GCP

### PR DESCRIPTION
**Pull request by sunix**

### CODE REVIEW

File: modules/installation-guide/partials/proc_configuring-the-che-operator-checluster-resource-with-kubectl.adoc [hunk 1]
```diff
@@ -0,0 +1,12 @@
+
+
+[id="configuring-the-che-operator-checluster-resource-with-kubectl_{context}"]
+
+= Configuring CheCluster Custom Resource with kubectl
+Edit the CheCluster custom resource configuration with kubectl:
```
Comment (yhontyk) [inline modules/installation-guide/partials/proc_configuring-the-che-operator-checluster-resource-with-kubectl.adoc:6]:
```suggestion
Edit the CheCluster custom resource configuration with {prod-cli}:
```

File: modules/installation-guide/partials/proc_configuring-the-che-operator-checluster-resource-with-kubectl.adoc [hunk 2]
```diff
@@ -0,0 +1,12 @@
+
+
+[id="configuring-the-che-operator-checluster-resource-with-kubectl_{context}"]
+
+= Configuring CheCluster Custom Resource with kubectl
```
Comment (yhontyk) [inline modules/installation-guide/partials/proc_configuring-the-che-operator-checluster-resource-with-kubectl.adoc:5]:
```suggestion
= Configuring CheCluster Custom Resource with {prod-cli}
```

File: modules/installation-guide/partials/proc_configuring-the-che-operator-checluster-resource-with-kubectl.adoc [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+Edit the CheCluster custom resource configuration with kubectl:
+
+----
+kubectl edit CheCluster/eclipse-che -n che
```
Comment (yhontyk) [inline modules/installation-guide/partials/proc_configuring-the-che-operator-checluster-resource-with-kubectl.adoc:10]:
```suggestion
$ {prod-cli} edit CheCluster/eclipse-che -n che
```

File: modules/installation-guide/partials/proc_sizing-google-cloud-node-pools-for-your-eclipse-che-workspaces.adoc [hunk 4]
```diff
@@ -0,0 +1,18 @@
+
+
+[id="sizing-google-cloud-node-pools-for-your-eclipse-che-workspaces_{context}"]
+= Sizing Google Cloud node pools for your Eclipse Che workspaces
+
+The default node pool is using machine types `n1-standard-1`. You may need to use different kind of machine to run your workspaces. See https://cloud.google.com/compute/docs/machine-types?hl=en#n1_standard_machine_types
```
Comment (yhontyk) [inline modules/installation-guide/partials/proc_sizing-google-cloud-node-pools-for-your-eclipse-che-workspaces.adoc]:
```suggestion
The default node pool is using machine types `n1-standard-1`. You may need to use different kind of machine to run your workspaces. See link:https://cloud.google.com/compute/docs/machine-types?hl=en#n1_standard_machine_types[]
```

File: modules/installation-guide/partials/proc_sizing-google-cloud-node-pools-for-your-eclipse-che-workspaces.adoc [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+
+One solution is to create a new node pool and delete the existing one afterwards. Google Kubernetes Engine would then transfer the workloads to the new node pool automatically.
+
+Another solution is to have two node pools with one dedicated to the workspaces by configuring {prod} with a NodeSelector. See xref:configuring-workspaces-nodeselector.adoc[].
+
+image::installation/gcp-new-nodepool.png[link="../_images/installation/gcp-new-nodepool.png"]
+
+image::installation/gcp-new-nodepool-select-machine-type-standard.png[link="../_images/installation/gcp-new-nodepool-select-machine-type-standard.png"]
+
+Preferably choose machine types according the workspaces CPU and memory requirements.
```
Comment (yhontyk) [inline modules/installation-guide/partials/proc_sizing-google-cloud-node-pools-for-your-eclipse-che-workspaces.adoc]:
```suggestion
Choose machine types according the workspaces CPU and memory requirements.
```

File: modules/installation-guide/partials/proc_configuring-the-che-operator-checluster-resource-with-kubectl.adoc [hunk 6]
```diff
@@ -0,0 +1,12 @@
+
+
+[id="configuring-the-che-operator-checluster-resource-with-kubectl_{context}"]
+
+= Configuring CheCluster Custom Resource with kubectl
+Edit the CheCluster custom resource configuration with kubectl:
+
```
Comment (yhontyk) [inline modules/installation-guide/partials/proc_configuring-the-che-operator-checluster-resource-with-kubectl.adoc]:
```suggestion
+
[subs="+attributes"]
```

**Review by yhontyk — COMMENTED** [review]
A few suggestions

**Comment by sunix** [comment]
hey sorry @boczkowska I have to revert some of the suggestions ... as it is `kubectl` and not `chectl` (prod-cli)

**Comment by sunix** [comment]
ok @boczkowska could you check if it is ok for you?

**Review by yhontyk — APPROVED** [review]

**Pull request merged**